rook/Dockerfile

```diff
@@ -22,7 +22,7 @@ WORKDIR ${ROOK_DIR}/images/ceph
 RUN make generate-csv-ceph-templates
 
 # Stage2: setup runtime container
-FROM ceph/ceph-amd64:v14.2.6-20200115
```

Comment:
This can be given by a command-line argument as shown in this article:
https://www.jeffgeerling.com/blog/2017/use-arg-dockerfile-dynamic-image-specification

It'd be better to parameterize the base image, especially, the image tag `:14.2` and supply
the value in CircleCI config.

Reply:
I think the info about what version to use should be separated from CircleCI config.
So, the base image name and its version should be written in each Dockerfile, shouldn't it?
(I know some exceptions already exist, e.g. envoy and ceph...

Reply:
I thought so because ceph version is specified in circleci config.

Reply:
That said, this is minor.  I'll mark this as resolved.